esp-hal-common/src/soc/esp32h2/mod.rs

```diff
@@ -0,0 +1,8 @@
+pub mod efuse;
+pub mod gpio;
+pub mod peripherals;
+pub mod radio_clocks;
+
+pub(crate) mod registers {
+    pub const INTERRUPT_MAP_BASE: u32 = 0x60010000; // FIXME
```

Comment:
Let's fix this please!

Reply:
Comment deleted, value is already correct